media/manuscript-markdown.css

```
@@ -119,3 +119,61 @@ body.vscode-dark .manuscript-markdown-highlight-gray-50 { background-color: rgba
+}
+.markdown-alert > p:empty {
+  display: none;
+}
+.markdown-alert-title {
+  display: flex;
+  align-items: center;
+  gap: 0.5rem;
+  font-weight: 600;
+}
+.markdown-alert-icon {
+  fill: currentColor;
```

Stylelint (17.3.0)

[error] Expected "currentColor" to be "currentcolor" (value-keyword-case)

(value-keyword-case)